### DISCUSSION TITLE
Using multiple models in calendar

### DISCUSSION THREAD

**Discussion by woutermoelker** [saade/filament-fullcalendar#157] [Q&A]
Hi there,

Im making a calendar that has to use multiple models, availability and appointment, both have to be editable, and viewable, but theyre using different models. Right now the model is initialized at `public Model | string | null $model = Availability::class;`
But i cant seem to figure out how i could utilize two models in the same calendar. Does somebody know the solution to this?

Thanks!

![Screenshot 2024-02-12 at 13 37 13](https://github.com/saade/filament-fullcalendar/assets/7361523/f6271086-0ac2-459d-b03a-fca16b4248d3)

**Comment by cawecoy** [comment]
Hi @woutermoelker! Have you found a solution?

**Reply by cawecoy** [reply]
I found a possible solution here by chance: https://github.com/saade/filament-fullcalendar/issues/140#issuecomment-1914429077

**Comment by LeamonaDor1** [comment]
@woutermoelker 
Do you want to display two models **simultaneously** on the same calendar or make the calendar dynamic so that it can display events from any model?
